cmd/account/cli.go

```diff
@@ -36,62 +27,41 @@ func newCmdCli(streams genericclioptions.IOStreams, flags *genericclioptions.Con
 		},
 	}
 
-	cliCmd.Flags().StringVar(&ops.accountNamespace, "account-namespace", common.AWSAccountNamespace,
-		"The namespace to keep AWS accounts. The default value is aws-account-operator.")
-	cliCmd.Flags().StringVarP(&ops.accountName, "account-name", "a", "", "The AWS Account CR name to generate the credentials for")
-	cliCmd.Flags().StringVarP(&ops.accountID, "account-id", "i", "", "The AWS Account ID we need to create temporary AWS credentials for -- This argument will not work for CCS accounts")
-	cliCmd.Flags().StringVarP(&ops.clusterID, "cluster-id", "C", "", "The Internal Cluster ID from Hive to create AWS console URL for")
-	cliCmd.Flags().StringVarP(&ops.profile, "aws-profile", "p", "", "specify AWS profile")
-	cliCmd.Flags().StringVarP(&ops.cfgFile, "aws-config", "c", "", "specify AWS config file path")
-	cliCmd.Flags().StringVarP(&ops.region, "aws-region", "r", common.DefaultRegion, "specify AWS region")
-	cliCmd.Flags().Int64VarP(&ops.cliDuration, "duration", "d", 3600, "The duration of the cli token. "+
-		"Default value is 3600 seconds(1 hour)")
+	ops.k8scloudfactory.AttachCobraCliFlags(cliCmd)
+
 	cliCmd.Flags().BoolVarP(&ops.verbose, "verbose", "v", false, "Verbose output")
 
 	return cliCmd
 }
 
 // cliOptions defines the struct for running the cli command
 type cliOptions struct {
-	accountName      string
-	accountID        string
-	accountNamespace string
-	clusterID        string
-	cliDuration      int64
-
-	// AWS config
-	region  string
-	profile string
-	cfgFile string
+	k8scloudfactory k8spkg.ClusterResourceFactoryOptions
 
 	verbose bool
 
-	flags *genericclioptions.ConfigFlags
 	genericclioptions.IOStreams
-	kubeCli client.Client
 }
 
 func newCliOptions(streams genericclioptions.IOStreams, flags *genericclioptions.ConfigFlags) *cliOptions {
 	return &cliOptions{
-		flags:     flags,
+		k8scloudfactory: k8spkg.ClusterResourceFactoryOptions{
```

Comment:
This parameter is still named `cloudfactory`, shouldn't it be also renamed to `clusterresourcefactory`?